core/services/keystore/ocr2.go

```diff
@@ -20,7 +20,7 @@ type OCR2 interface {
 	Delete(id string) error
 	Import(keyJSON []byte, password string) (ocr2key.KeyBundle, error)
 	Export(id string, password string) ([]byte, error)
-	EnsureKeys() error
+	EnsureKeys(enabledChains chaintype.ChainTypes) error
```

Comment:
I was thinking that variadic would make sense here, but I guess it would only make the test code more clear in this case :thinking: 
```suggestion
	EnsureKeys(enabledChains ...chaintype.ChainType) error
```